<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DicomNiftiModule</class>
 <widget class="qMRMLWidget" name="DicomNiftiModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>328</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QCheckBox" name="autoUploadCheckBox">
       <property name="text">
        <string>Auto Upload</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="conversionLabel">
       <property name="text">
        <string>Conversion Progress:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QPushButton" name="uploadButton">
       <property name="text">
        <string>Upload to Server</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="ctkPathLineEdit" name="outputFolderSelector">
       <property name="filters">
        <set>ctkPathLineEdit::Dirs|ctkPathLineEdit::Drives|ctkPathLineEdit::NoDot|ctkPathLineEdit::NoDotDot|ctkPathLineEdit::Readable</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="uploadLabel">
       <property name="text">
        <string>Upload Progress:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="studiesFolderLabel">
       <property name="text">
        <string>Studies Folder Path</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <widget class="QTextBrowser" name="logTextBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="dicomFolderLabel">
       <property name="text">
        <string>Select DICOM Folder</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="outputFolderLabel">
       <property name="text">
        <string>Select Output Folder (Optional)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="ctkPathLineEdit" name="inputFolderSelector">
       <property name="filters">
        <set>ctkPathLineEdit::Dirs|ctkPathLineEdit::Drives|ctkPathLineEdit::NoDot|ctkPathLineEdit::NoDotDot|ctkPathLineEdit::Readable</set>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="QProgressBar" name="uploadProgressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="ctkPathLineEdit" name="studiesFolderSelector"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="logLabel">
       <property name="text">
        <string>Log</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QPushButton" name="convertButton">
       <property name="text">
        <string>Convert to NIfTI</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QProgressBar" name="conversionProgressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QCheckBox" name="skipExistingCheckBox">
       <property name="text">
        <string>Skip Existing Files (Automatically skip duplicates)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
